app/init.php

```diff
@@ -724,6 +724,18 @@ function (mixed $value) {
     $providerName = System::getEnv('_APP_LOGGING_PROVIDER', '');
     $providerConfig = System::getEnv('_APP_LOGGING_CONFIG', '');
 
+    try {
+        $loggingProvider = new DSN($providerConfig ?? '');
+
+        $providerName = $loggingProvider->getScheme();
+        $providerConfig = match ($providerName) {
+            'sentry' => ($loggingProvider->getUser() ?? '') . ';' . $loggingProvider->getHost(),
+            default => $loggingProvider->getHost(),
+        };
```

Comment:
https://github.com/utopia-php/logger/pull/37 

We changed the sentry & other constructors a bit to accept the parameters explicitly. We might want to adapt this PR to those changes.